docs/api/covidcast-signals/indicator-combination.md

```diff
@@ -46,6 +50,158 @@ coronavirus infection---to produce a single indicator.
+this suffers from the issue that the locally scaled time series for a given
+sensor and region can look quite different from the unscaled time series. In
+particular, increasing trends can be replaced with decreasing trends and, as a
+result, the derived combined indicator may look quite distinct from any unscaled
+sensor. This can be avoided with global column scaling.
+
+#### Lags and Sporadic Missingness
+
+The matrix $$X$$ is not necessarily complete and we may have entries missing.
+Several forms of missingness arise in our data.
+
+* On certain days, all observations of a given sensor are missing due to release
+  lag. For example, Doctor Visits is released several days late.
+* On any given day, different sensors are observed in different regions.
+* For any given region and sensor, the sensor may be available on some days but
+  not others due to sample size cutoffs.
+
+To ensure that our combined indicator value has comparable scaling over time and
+is free from erratic jumps that are just due to missingness, we use the
+following strategies:
+
+* *Lag imputation*: If a sensor is missing for all regions on a given day, we
+  copy all observations from the last day on which any observation was available
+  for that sensor.
+* *Recent imputation*: If $$x_{rs}(t)$$ is missing but at least one of
+  $$x_{rs}(t-1), x_{rs}(t-2), \dots, x_{rs}(t-T)$$ is observed, impute
+  $$x_{rs}(t)$$ with the most recent of $$x_{rs}(t-1), x_{rs}(t-2), \dots,
+  x_{rs}(t-L)$$. We limit $$T$$ to be 7 days.
+
+#### Persistent Missingness
+
+Even with the above imputation strategies, we still have issues that some
+sensors are never available in a given region. The result is that combined
+indicator values for that region that may be on a completely different scale
+from values in other regions with additional observed sensors. This can only be
+overcome by regularizing / pooling information across space. Note that a very
+similar problem occurs when a sensor is unavailable for a very long period of
+time (so long that recent imputation is inadvisable and avoided by setting $$T =
+7$$ days).
+
+We deal with this problem by *geographic imputation* where we impute values from
+regions that share a higher level of aggregation (e.g., the mean observed score
+in an MSA / state), or by imputing values from megacounties (since the counties
+in question are missing and hence should be reflected in the rest of state
+estimate). The order in which we try to perform geographic imputations is first
+attempting to impute from observed values from megacounties, followed by median
+observed values in the geographic hierarchy (county, MSA, and state), followed
+finally by the median observed value over the entire country if all the previous
+imputation attempts fail. This imputation order is chosen among different
+options by evaluating their effectiveness to mimic the actual observed values in
+validation experiments.
```

Comment:
```suggestion
We deal with this problem by *geographic imputation*, where we impute values from
regions that share a higher level of aggregation (e.g., the median observed score
in an MSA or state), or by imputing values from megacounties (since the counties
in question are missing and hence should be reflected in the rest of state
estimate). The order in which we look to perform geographic imputations is
observed values from megacounties, followed by median
observed values in the geographic hierarchy (county, MSA, state, or country).
We chose this imputation sequence among different options by evaluating 
their effectiveness to mimic the actual observed sensor values in validation experiments.
```